### DISCUSSION TITLE
Starting with extensions, failing on the simple example example

### DISCUSSION THREAD

**Discussion by coignite-user** [directus/directus#7067] [Archive]
I followed the guide here: https://docs.directus.io/guides/modules/ and ran into a problem when building the module.

This is probably something very simple, but I tried different options with the same result, directus-extension CLI telling me
```
[Error] Current directory is not a Directus extension.
```
![image](https://user-images.githubusercontent.com/28866431/127461952-ee95c05f-c36b-4c22-b6fb-84222ec98146.png)

What can be a problem here?

**Comment by coignite-user** [comment]
I tried to run
```
npx directus-extension create module dashboard
```
inside `src` folder, but it only creates `dashboard` folder and then runs continiously without doing anything, probably because of
```
(node:49197) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, lstat '/Volumes/Data/Code/directus-app-platform-pure/node_modules/@directus/extension-sdk/templates/common'
```

**Comment by rijkvanzanten** [comment]
See https://github.com/directus/directus/discussions/6970#discussioncomment-1052186 🙂
